search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -331,6 +331,18 @@
                                     not-empty)]
      (str (string/join ". " citation-details) "."))))
 
+(defn- nil-if-not-provided
+  "nil if not provided."
+  [s]
+  (when-not (= umm-spec-util/not-provided s)
+    s))
+
+(defn graphic-preview-type
+  "Use format or mime type if provided."
+  [related-url]
+  (or (nil-if-not-provided (:format related-url))
```

Comment:
Isn't the open data format looking for a MIME type? 

If so - Is Format already a MIME type, or do we need to do some conversion from the Format string to a MIME type?

Similarly I'm thinking that MIME type should also be our first choice if open data is looking for a MIME type here.

@eereiter - what is your take?

Reply:
The preview type doesn't matter. It will display on data.gov based off the file extension i believe. The graphic-preview-type is just extra information.

Reply:
Yes , I agree.  Format is not the MimeType.  For example JPEG could be the format, but the mime type would be image/jpeg. Most of the time there is a direct mapping, but not all of the time.

Reply:
So for the graphic-preview-type would the mime-type be the correct to display?

Reply:
yes